### ISSUE TITLE
Distributor Module doesn't select targets.

### ISSUE THREAD

**Issue by Kazuhiko-Gushiken** [desht/ModularRouters#235]
### Describe the bug

Distributor Module doesn't select targets. It just keeps saying target added 1/8 but doesnt add anything.

### How to reproduce the bug

Shift right click an inventory with the distributor module

### Expected behavior

It to select blocks

### Which Minecraft version are you using?

1.21

**Comment by darklotus781** [comment]
Same issue here, the distributor module claims to add 1/8 targets but then won't add others and it doesn't highlight.
Existing distributors are working still.
After using a new distributor to select a target, it causes other issues like doors opening on top and not bottom.  Have to relog or restart the server to resolve.
Not sure if it's helpful but this is what's logged on the server after attempting to use the module.
```
[20:40:18] [Server thread/ERROR] [minecraft/ServerPacketListener]: Failed to handle packet net.minecraft.network.protocol.game.ServerboundUseItemOnPacket@253283fe, suppressing error
java.lang.IllegalStateException: Cannot get config value before config is loaded.
        at MC-BOOTSTRAP/com.google.common@31.1-jre/com.google.common.base.Preconditions.checkState(Preconditions.java:502) ~[guava-31.1-jre.jar%2383!/:?] {re:mixin}
        at TRANSFORMER/neoforge@21.0.112-beta/net.neoforged.neoforge.common.ModConfigSpec$ConfigValue.get(ModConfigSpec.java:1171) ~[neoforge-21.0.112-beta-universal.jar%23268!/:?] {re:mixin,re:classloading}
        at TRANSFORMER/modularrouters@13.0.2/me.desht.modularrouters.item.module.TargetedModule.handleMultiTarget(TargetedModule.java:111) ~[modular-routers-13.0.2+mc1.21.jar%23369!/:?] {re:classloading}
        at TRANSFORMER/modularrouters@13.0.2/me.desht.modularrouters.item.module.TargetedModule.useOn(TargetedModule.java:58) ~[modular-routers-13.0.2+mc1.21.jar%23369!/:?] {re:classloading}
        at TRANSFORMER/neoforge@21.0.112-beta/net.neoforged.neoforge.common.CommonHooks.onPlaceItemIntoWorld(CommonHooks.java:610) ~[neoforge-21.0.112-beta-universal.jar%23268!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.world.item.ItemStack.useOn(ItemStack.java:360) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:placebo.mixins.json:ItemStackMixin from mod placebo,pl:mixin:APP:apothic_attributes.mixins.json:ItemStackMixin from mod apothic_attributes,pl:mixin:APP:kubejs.mixins.json:ItemStackMixin from mod kubejs,pl:mixin:APP:showcaseitem.mixins.json:ItemStackMixin from mod showcaseitem,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.level.ServerPlayerGameMode.useItemOn(ServerPlayerGameMode.java:386) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItemOn(ServerGamePacketListenerImpl.java:1122) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ftbquests-common.mixins.json:ServerGamePacketListenerImplMixin from mod ftbquests,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.handle(ServerboundUseItemOnPacket.java:42) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.handle(ServerboundUseItemOnPacket.java:10) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:27) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:148) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,re:computing_frames,re:classloading}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:872) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:170) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:855) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:849) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:821) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:826) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:712) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at TRANSFORMER/minecraft@1.21/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:267) ~[server-1.21-20240613.152323-srg.jar%23267!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?] {re:mixin}
```

**Comment by desht** [comment]
Confirmed, will be fixed in next release.

**Comment by desht** [comment]
Fixed in 13.0.3 release
